TFlippy_TerritoryControl_Structures_Dev/Entities/Structures/SpawnRuins/Ruins.cfg

```diff
@@ -50,7 +50,10 @@ bool shape_collides                           = no
 bool shape_ladder                          = no
 bool shape_platform                        = no
  #block_collider
-@f32 verticesXY                            = 
+@f32 verticesXY                                   = 0.0; 0.0;
+													40.0; 0.0;
+													40.0; 56.0;
+													0.0; 56.0;
```

Comment:
formatting is incorrect, not a big deal, just something to keep an eye out for in the future

Reply:
saying that tho, it looks like you copy pasted it from witch shack so thats fine